src/eulerevaluator.cpp

```diff
@@ -48,15 +48,27 @@ NetworkState EulerEvaluator::GetNextNetworkState() {
 			for (auto &reactant : term.second) {
 				change *= pow(oldState[reactant.first], reactant.second);
 			}
-			diff += change;
+			diff = KahanSummation({diff, change});
```

Comment:
Jeg tror måske du har misforstået Kahan's algoritme lidt. Ideen er at man under hele summeringen beholder `c`. Men ved at kalde den med to tal hver gang, er det faktisk ikke bedre end et normalt plus-stykke.